5minutes.html

```diff
@@ -28,8 +28,8 @@
                     <section>
                         <h1>
                             <img src="img/logo_code_for_poznan.png" style="background:none; border:none; box-shadow:none; "/>
-                            <h3>Girls.js</h3>
-                            <h4>03-04.03.2018, Poznań</h4>
+                            <h3>Django Girls Workshop #2</h3>
```

Comment:
Workshops (plural)

Reply:
(it shouldn't be plural, as its targeted at second workshop, not two of them)

Reply:
Disagree. ;-) 

I would stick with: `Django Girls Poznań #2` 

Workshops is used in official communication, there is no place with workshop in singular form. (or at least I did not see)